### PR TITLE
Updated meetups for Aug-Sep 2016

### PR DESCRIPTION
**Pull request by asgrim**
Yes...

### CODE REVIEW

File: data/meetups/2016-08-10.php [hunk 1]
```diff
@@ -31,15 +31,15 @@
     ->setLocationUrl("https://www.google.co.uk/maps?q=Oasis+Venue,+Arundel+Street,+PO1+1NP&hl=en&ll=50.799642,-1.086724&spn=0.011772,0.031629&sll=50.799734,-1.086874&sspn=0.011772,0.031629&hq=Oasis+Venue,&hnear=Arundel+St,+PO1+1NP,+United+Kingdom&t=m&z=16")
     ->setLocation('Oasis the Venue, Arundel Street, PO1 1NP')
     ->setTalkingPoints(array(
-        'Lightning talk - tbc',
+        'Lightning talk - Andrew Lucas',
```
Comment (lboynton) [inline data/meetups/2016-08-10.php]: What no Twitter?

File: data/meetups/2016-09-14.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+END;
+
+$meetup->setId(0)
+    ->setFromDate(new DateTime('2016-09-14 19:00'))
+    ->setToDate(new DateTime('2016-09-14 23:00'))
+    ->setRegistrationUrl("https://www.eventbrite.co.uk/event/{$eid}")
+    ->setLocationUrl("https://www.google.co.uk/maps?q=Oasis+Venue,+Arundel+Street,+PO1+1NP&hl=en&ll=50.799642,-1.086724&spn=0.011772,0.031629&sll=50.799734,-1.086874&sspn=0.011772,0.031629&hq=Oasis+Venue,&hnear=Arundel+St,+PO1+1NP,+United+Kingdom&t=m&z=16")
+    ->setLocation('Oasis the Venue, Arundel Street, PO1 1NP')
+    ->setTalkingPoints(array(
+        'Lightning talk - Richard Harrison',
```
Comment (lboynton) [inline data/meetups/2016-09-14.php]: Richard again?

Comment (asgrim) [inline data/meetups/2016-09-14.php]: Yup, he offered, I'm not gonna turn it down ;)

Comment (lboynton) [inline data/meetups/2016-09-14.php]: No, he is a good speaker :)

**Pull request merged**